meta-resin-common/recipes-connectivity/openvpn/openvpn/prepare-openvpn

```diff
@@ -12,19 +12,14 @@ if [ ! -f $CONFIG_PATH ]; then
 	exit 1
 fi
 
-while :; do
-	_uuid=$(jq -r '.uuid' $CONFIG_PATH)
-	# If the user api key exists we use it instead of the deviceApiKey as it means we haven't done the key exchange yet
-	_device_api_key=$(jq -r '.apiKey // .deviceApiKey' $CONFIG_PATH)
+# If the user api key exists we use it instead of the deviceApiKey as it means we haven't done the key exchange yet
+_device_api_key=${PROVISIONING_API_KEY:-DEVICE_API_KEY}
```

Comment:
Isn't the plan to make the provisioning key only able to provision and therefore not connect to vpn? Shouldn't this then only check for device  apikey?